charts/spire/charts/spire-server/values.yaml

```diff
@@ -220,23 +218,23 @@ nodeAttestor:
 
 # tornjak - Tornjak specific configuration
 tornjak:
+  # -- Deploys Tornjak API (backend)
   enabled: false
-  # image - Tornjak image (frontend + backend) if not separated above
-  image: # ghcr.io/spiffe/tornjak
+  # -- Tornjak API image
+  image:
     registry: ghcr.io
-    repository: spiffe/tornjak
+    repository: spiffe/tornjak-be
     pullPolicy: IfNotPresent
-    # Overrides the image tag whose default is the chart appVersion.
-    # TODO we should use a specific Tornjak version instead of 'latest'
-    version: "latest"
+    # -- Overrides the image tag whose default is the chart appVersion.
```

Comment:
If you have it specified as below (which is needed), then the comment is incorrect as it will never be appVersion.

Reply:
what name would you have in mind? 
Would this work? 
```
persistenceDB:
    driver: "sqlite3"
    file: "/file/name"
```

Reply:
just 'persistence' rather then persistenceDB. kind of a helm convention.

Though this seems to be generating a dataStore section in a config file and not related to an actual pvc creation? maybe dataStore is the right thing in that case.... 

yeah... https://github.com/spiffe/helm-charts/blob/tornjak/charts/spire/charts/spire-server/templates/tornjak-config.yaml#L13-L20

ok. I think probably leaving it as .Values.tornjak.dataStore is the right thing.

Reply:
Thank you. I think that is the best outcome. If PVC is available it can be used, otherwise it's just a dataStore.